algorithms/Cargo.toml

```diff
@@ -127,6 +127,12 @@ features = [ "use_core" ]
 [dependencies.digest]
 version = "0.10"
 
+[dependencies.getrandom]
```

Comment:
Should this be a dev-dependency?

Reply:
WASM requires this feature in order to instantiate an Rng